service_container/tags.rst

```diff
@@ -51,14 +56,10 @@ to be used for a specific purpose. Take the following example:
             ->setPublic(false)
             ->addTag('twig.extension');
 
-The ``twig.extension`` tag is a special tag that the TwigBundle uses
-during configuration. By giving the service this ``twig.extension`` tag,
-the bundle knows that the ``app.twig_extension`` service should be registered
-as a Twig extension with Twig. In other words, Twig finds all services tagged
-with ``twig.extension`` and automatically registers them as extensions.
-
-Tags, then, are a way to tell Symfony or other third-party bundles that
-your service should be registered or used in some special way by the bundle.
+Services tagged with the ``twig.extension`` tag are collected during the
+initialization of TwigBundle to enable them as Twig extensions. In this example,
+that's why Symfony knows that the ``app.twig_extension`` service should be
+registered as a Twig extension.
```

Comment:
I'm not convinced this last sentence adds any value.

A total alternative, could be:

> Other tags are used to integrate your services into *other* systems.

(actually, this could be the first sentence of the next paragraph)